docs/source/NEXUS.rst

```diff
@@ -33,10 +33,7 @@ Important information about NEXUS code:
 Tags
 ------------
 A list of the different NEXUS `tags <https://github.com/next-exp/nexus/wiki/Tags>`_ can be found on the Wiki.
-Starting from nexus tag v7_00_00, the following versions are needed:
-
- * Geant4 version >= 11.0.
- * ROOT >= 6.26.
+Starting from nexus tag v7_00_00, Geant4 version >= 11.0 is needed; for previous nexus tags,  ROOT >= 6.26 must be used.
```

Comment:
Which Geant version should be used before v7?

Reply:
Not sure. Versions 11 and 10 should work (except for 10.7.x, as specified below), but before that I don't remember.

Reply:
In the blue box we just say that 10.5 or 10.6 can be used.